packages/shorebird_code_push_protocol/lib/src/messages/get_usage/get_usage_response.dart

```diff
@@ -46,6 +47,16 @@ class GetUsageResponse {
   final int? patchInstallLimit;
 }
 
+/// Converts [currentPeriodCost] to a [Money] object.
+extension GetUsageResponseMoney on GetUsageResponse {
+  /// [currentPeriodCost] as a [Money] object. Prefer this over using
+  /// [currentPeriodCost] directly.
+  Money get currentPeriodCostMoney => Money.fromIntWithCurrency(
+        currentPeriodCost,
```

Comment:
We could rename it, but I don't think it's possible to hide. I opted to leave it as-is to avoid making a breaking change. @felangel open to your thoughts on this.